### ISSUE TITLE
"There were no sales for the date specified" from download_sales_and_trends_reports for subscription

### ISSUE THREAD

**Issue by anatoly-kor** [Ponytech/appstoreconnectapi#54]
I have strange behavior: "no sales for the date specified" after 01 March for subscription, but sales is exists and lib show data. I tried to make request directly to API and subscriptions is exists.

**Comment by gavr** [comment]
There is new version of API https://developer.apple.com/news/upcoming-requirements/?id=03012023a

**Comment by DevMike911** [comment]
how to fix this issue? just rename 1.2 --> 1.3?

**Comment by gavr** [comment]
> how to fix this issue? just rename 1.2 --> 1.3?

Yep, you can use example https://github.com/Ponytech/appstoreconnectapi/pull/55/files
